core/src/main/java/org/jdbi/v3/core/Handle.java

```diff
@@ -322,10 +325,14 @@ public Handle rollback() {
     public Handle rollbackToSavepoint(String savepointName) {
         final long start = System.nanoTime();
         transactions.rollbackToSavepoint(this, savepointName);
-        LOG.trace("Handle [{}] rollback to savepoint \"{}\" in {}ms", this, savepointName, (System.nanoTime() - start) / 1000000L);
+        LOG.trace("Handle [{}] rollback to savepoint \"{}\" in {}ms", this, savepointName, msSince(start));
         return this;
     }
 
+    private static long msSince(final long start) {
```

Comment:
I feel we're better off making a `Stopwtch` class if timing comes up in multiple places

Reply:
I think it's clearer to have a simple one line function in the local file, rather than to add a whole new file.
I'm not opposed to a stopwatch class but I'd recommend we do that as a separate PR that considers the consequences properly rather than just on a whim in a cleanup.

Reply:
Could be as simple as this IMO:
```
public class Timer {
    private Instant start;
    
    public Timer start() {
        start = Instant.now();
        return this;
    }
    
    public long measure(ChronoUnit unit) {
        return unit.between(start, Instant.now());
    }
}
```

Reply:
My point is that it adds a non-local reference for the reader when we can sensibly keep it local for now.  This PR isn't intended to add a generalizable Stopwatch.